### DISCUSSION TITLE
Tuya TS0014 WITH neutral

### DISCUSSION THREAD

**Discussion by oztourer** [Koenkk/zigbee2mqtt#11314] [Devices]
Hi,

I am new to zigbee2mqtt and zigbee in general, so forgive me if there is an easy answer here.

I bought a number of unbranded wall switches (1, 2 and 4-gang) from [ozsmartthings](https://www.ozsmartthings.com.au/collections/zigbee). On pairing with zigbee2mqtt they turn out to be rebranded Tuya devices and come up as TS0011/_TYZB01_8ppvdbpz, TS0012/_TYZB01_6g8b7at8 and [TS0014/_TYZB01_bagt1e4o](https://www.zigbee2mqtt.io/devices/TS0014.html). All show in the zigbee2mqtt map as endpoints, not routers, despite being wired with live and neutral.

Neutral is optional on these switches but I have wired with neutral with the expectation that they would then act as routers. I have been told that they do function as routers when paired with a Tuya gateway, and I am relying on that to achieve a workable mesh network.

I tried using zigbee2mqtt edge but that made no difference.

Is there any way to override zigbee2mqtt's default config so that they will be seen as routers? Any advice would be much appreciated.

**Comment by JoryHogeveen** [comment]
No use in registering these devices are routers simply because they are not routers. The TS001X line are end devices and the TS000X line are routers.

Edit: The manufacturers of these type of devices aren't really clear/transparent as to what kind of Zigbee chip they are using. I honestly doubt their support staff even knows...

Related to this issue I started the following topic on the community: https://community.home-assistant.io/t/user-experience-request-differernt-zigbee-tuya-based-touch-light-switches/439249
Any contributions to experience with these devices are very welcome!
